providers/linode/linodeProvider_test.go

```diff
@@ -0,0 +1,22 @@
+package linode
+
+import (
+	"testing"
+)
+
+func TestFixTTL(t *testing.T) {
```

Comment:
Please add a unit test for {299, ???} (i.e. that way you are testing, n, n-1, and n+1)

(by the way... thank you so much for having unit tests. So few people write them and they are very important for keeping the code maintainable)